Tests/SwiftDocCTests/Semantics/SymbolTests.swift

```diff
@@ -485,7 +485,7 @@ class SymbolTests: XCTestCase {
             docComment: """
                 A cool API to call.
 
-                @Redirected(from: "some/previous/path/to/this/symbol")
+                @Image returns known directive
```

Comment:
I don't think we should need to change this test to make it pass.

Are we just missing the `Redirect` directive in the `allKnownDirectives` list?

https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Semantics/Redirect.swift

Reply:
thats right, I added @Image for it to have a known directive. I see the list i have is a subset of what your mentioned below https://github.com/apple/swift-docc/pull/154#discussion_r860352906. My criteria for finding known Directive was different and i see many were missed when i searched by "public static let directiveName"

Reply:
Should we revert this change now?

Reply:
Yes, i agree.